tests/bam/bam_pb.robot

```diff
@@ -755,7 +755,7 @@ BEPB_DIMENSION_BA_TIMEPERIOD_RELATION
 	Kindly Stop Broker  True
 
 BEPB_DIMENSION_TRUNCATE_TABLE
-	[Documentation]	use of pb_dimension_timeperiod message.
+	[Documentation]	use of pb_dimension_timeperiod Removing lua file before starting, creating a BA and adding lua output to broker waiting for the lua file to be created again and searching for a specific event in the log file using the Grep File keyword, the event should have a certain type, category, element, and update_started status.
```

Comment:
here too, the lua script is just to help us to see what events arrived into broker. It is called each time an event is received by broker and the lua script logs it into a file.
Don't speak too much about this script.